cli/tokens.go

```diff
@@ -233,7 +233,11 @@ func (r *RootCmd) removeToken() *serpent.Command {
 		Handler: func(inv *serpent.Invocation) error {
 			token, err := client.APIKeyByName(inv.Context(), codersdk.Me, inv.Args[0])
 			if err != nil {
-				return xerrors.Errorf("fetch api key by name %s: %w", inv.Args[0], err)
+				err = client.DeleteAPIKey(inv.Context(), codersdk.Me, inv.Args[0])
```

Comment:
If there's no API key by ID endpoint this should probably only trigger when the ID is the exact right length to avoid additional 400s/404s when people enter a name that doesn't exist